src/pages/home/ReportScreen.js

```diff
@@ -174,8 +194,11 @@ class ReportScreen extends React.Component {
                     nativeID={CONST.REPORT.DROP_NATIVE_ID}
                     style={[styles.flex1, styles.justifyContentEnd, styles.overflowHidden]}
                 >
-                    <FullScreenLoadingIndicator visible={this.shouldShowLoader()} />
-                    {!this.shouldShowLoader() && (
+                    {
+                      this.state.isGhostScreenVisible && <ChatGhostUI />
+                    }
+                    <FullScreenLoadingIndicator visible={!this.state.isGhostScreenVisible && this.shouldShowLoader()} />
```

Comment:
There are two loaders. :scream: ..... @shawnborton Could you please review the design?

Reply:
It is used for scrolling as well as if the report data already exists. Hence I've retained. I can remove the other one if required based on the feedback

Reply:
@shawnborton Can you confirm if we should be using only 1 loader? 

We've got three cases:
- Loader while API is being called
- Data already exists and switching between chats
- Load more (scroll up)

Reply:
I think it makes sense to see the ghost UI when you first load a chat - that could be when the app first loads, or when you switch between chats. 

I looked at Slack and it looks like they use the ghost UI for scrolling up too but it's not the smoothest thing. I'd say focus on the first cases of loading and switching chats first, and then if we can get this working smoothly when scrolling up, let's do it. I think my concern with using this when scrolling up is that we'll get some jumpy UI when the ghost UI goes in and out of the view given that the ghost UI doesn't use the exact same line heights/etc as the messages that would then load. Let me know if that makes sense!